src/lib/storage/fake_device_factory.rb

```diff
@@ -224,6 +160,7 @@ def create_disk(parent, args)
       def create_partition_table(parent, args)
         log.info("#{__method__}( #{parent}, #{args} )")
         disk_name = parent
+        args = "msdos" if args.downcase == "ms-dos" # Allow different spelling
```

Comment:
I fear this is now a little bit hidden here. I would look for a way to include this equivalence in `EnumMapping` in a more declarative way or would drop support for it. I don't like the idea of having some `if` here and there in addition to the mixin.